lib/agent/timer.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+function Timer (task, interval) {
+  this._handle = undefined
```

Comment:
I tend to define every class property in the constructor for documentation purposes.